docs/getting-started.md

```diff
@@ -157,49 +185,56 @@ excludes:
 ### Tips for Working with the `.ort.yml` File
 
 - Symlink the `ort.yml` created by `orthw` to the `.ort.yml` in the project code repository so you can commit your changes easily and often
-- Do not bother manually sorting path or scope excludes entries in the `ort.yml` file, simply add them and use `orthw rc-sort` to sort the file
+- Do not sort path or scope excludes entries in the `ort.yml` file manually, but simply add them and use `orthw rc-sort` to sort the file
```

Comment:
I'd like to mention another point:

Always sort the entries before modifying the `ort.yml` using `orthw` commands in order to minimize the diff produced by these commands. The commands are designed such that the diff does only one thing at a time and can be used to create a commit according to best practices (do one thing at a time), given the entries are sorted before.

Reply:
Frank, I can add this, but your text is not clear to me:

Do you mean users should run `orthw rc-sort` before they do anything that could change `.ort.yml` and then again after they've run some other command that may modify `.ort.yml`? 

Do we need to mention the `diff` -- at this point the user may have no idea that a diff is generated and what it is or how it is used. 

When you say, "The commands are designed such that the diff does only one thing at a time and can be used to create a commit according to best practices (do one thing at a time), given the entries are sorted before," do you mean the orthw commands work better if the '.ort.yml' entries are sorted correctly, because they are then likely to produce a smaller change set? With the implication that a smaller change is easier to understand and verify?

Reply:
Hm, agree my text is not clear. Shall we disregard it for the PR and then later on I can file a PR to add it on top? Then we can discuss it there. It's kind of out of scope here.

Reply:
I agree, Frank.